thrift/src/main/java/com/linecorp/armeria/server/thrift/THttpService.java

```diff
@@ -449,12 +449,20 @@ private SerializationFormat determineSerializationFormat(HttpRequest req) {
         if (contentType != null) {
             serializationFormat = findSerializationFormat(contentType);
             if (serializationFormat == null) {
-                return null;
+                // Browser clients often send a non-Thrift content type.
+                // Choose the default serialization format for some vague media types.
+                if (!("text".equals(contentType.type()) &&
+                      "plain".equals(contentType.subtype())) &&
+                    !("application".equals(contentType.type()) &&
+                      "octet-stream".equals(contentType.subtype()))) {
```

Comment:
This pull request revealed a bug which made the `contentTypeNotThrift()` test does not test anything.